pom.xml

```diff
@@ -432,6 +434,12 @@ limitations under the License.
                     <plugin>
                         <groupId>com.mycila</groupId>
                         <artifactId>license-maven-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/resources/**/*.sql</exclude>
+                                <exclude>**/docker/**/*.sql</exclude>
+                            </excludes>
+                        </configuration>
```

Comment:
Surely this doesn't need to be listed twice?  Nothing else had to be listed twice.

Reply:
@thilagasanthosh Have you addressed this comment yet?

Reply:
@michael-bryson , As discussed in yesterday's standup based on Dermot comments, the exclusion added to all SQL files. Now the duplicate listing is removed. 
All SQL files exclusion already tested with settings-service and it was working fine.

Reply:
@thilagasanthosh Ok regarding the excluding all sql files.
However the duplicate listing is still there:
- https://github.com/CAFapi/caf-common-parent/blob/develop/pom.xml#L313
- https://github.com/CAFapi/caf-common-parent/blob/develop/pom.xml#L438
Why is there 2 license profiles?

Reply:
the change is done in another PR and working fine in settings-service

https://github.com/CAFapi/caf-common-parent/pull/7